pyxem/generators/vdf_generator.py

```diff
@@ -27,7 +27,7 @@
 import numpy as np
 
 class VDFGenerator():
-    """Generates an VDF images for a specified signal and set of aperture
+    """Generates a VDF images for a specified signal and set of aperture
```

Comment:
Need to update the docstring below to reflect vectors being optional now, i.e.:

vectors : DiffractionVectors (optional)